v1/services/storage/store.go

```diff
@@ -30,8 +30,7 @@ type TSDBStore interface {
 	ShardGroup(ids []uint64) tsdb.ShardGroup
 	Shards(ids []uint64) []*tsdb.Shard
 	TagKeys(ctx context.Context, auth query.Authorizer, shardIDs []uint64, cond influxql.Expr) ([]tsdb.TagKeys, error)
-	TagValues(ctx context.Context, auth query.Authorizer, shardIDs []uint64, cond influxql.Expr) ([]tsdb.TagValues, error)
-	SeriesCardinality(ctx context.Context, database string) (int64, error)
```

Comment:
The `SeriesCardinality` method is not used from this interface - it should have been removed in https://github.com/influxdata/influxdb/pull/22441 but wasn't.